vimode/src/cmds/undo.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 Jiri Techet <techet@gmail.com>
```

Comment:
Your file, your name :-). You can skip the email address if you are worried of spam.